openstreetmap-carto.lua

```diff
@@ -14,6 +14,7 @@ local polygon_keys = {
     'building',
     'building:part',
     'harbour',
+    'healthcare',
```

Comment:
I think this might be harmful - did you try first importing data with this code and then rendering? I suppose every healthcare object will be no longer in hstore column then, so the whole database code should be different.

We did not such change after v4.0.0, and it should be announced much earlier and coordinated with forks and OSMF admins.

Reply:
Fixed, I removed the variable from the Lua script.